feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -112,6 +116,39 @@ import kotlinx.coroutines.flow.onCompletion
 private const val TAG = "PreviewScreen"
 private const val BLINK_TIME = 100L
 
+@Composable
+fun ElapsedTimeText(
+    modifier: Modifier = Modifier,
+    videoRecordingState: VideoRecordingState,
+    elapsedNs: Long
```

Comment:
noted.
I can just put this into a TODO until the VideoRecordingState refactor is complete. I can add the change to whichever PR is merged last.

Since the visibility depends on the `VideoRecordingState`, I don't want to add improper code to the `PreviewViewModel` that will just be deleted once the other PR is up. 
(I would need to do a little gymnastics b/c of line 154 in `PreviewViewModel`)

I also think that the objects in preview and quicksettings should eventually be refactored to have their own uiStates.